README.md

```diff
@@ -1 +1,86 @@
-# foresight-test-kit-action
\ No newline at end of file
+# Foresight Test Kit Action
+
+A GitHub Action to analyze test and/or coverage results.
+
+## Code in Main
+
+> First, you'll need to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
+
+Install the dependencies  
+```bash
+$ npm install
+```
+
+Build the typescript and package it for distribution
+```bash
+$ npm run all
+```
+
+Run the tests :heavy_check_mark:  
+```bash
+$ npm test
+```
+
+## Publish to a distribution branch
+
+Actions are run from GitHub repos so we will checkin the packed dist folder. 
+
+Then run [ncc](https://github.com/zeit/ncc) and push the results:
+```bash
+$ npm run package
+$ git add dist
+$ git commit -a -m "prod dependencies"
+$ git push origin releases/v1
+```
+
+Note: We recommend using the `--license` option for ncc, which will create a license file for all of the production node modules used in your project.
+
+Your action is now published! :rocket: 
+
+See the [versioning documentation](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)
+
+
+## Configuration
+
+| Option                | Requirement       | Description
+| ---                   | ---               | ---
+| `github_token`        | Optional          | An alternative GitHub token, other than the default provided by GitHub Actions runner.
+| `test_framework`      | Optional          | Runtime test framework name(jest, pytest, junit etc.)
+| `test_path`       | Optional              | Test results directory/file path.
+| `coverage_framework`      | Optional          | Runtime coverage framework name(jacoco etc.)
```

Comment:
`coverage_format` might be better name because in fact we are integrating with coverage file formats, not the coverage frameworks.
/cc @yusuf-erdem

Reply:
fixed.